gyp_xwalk

```diff
@@ -351,6 +351,10 @@ if __name__ == '__main__':
     args.append('-Dnotifications=1')
     args.append('-Drelease_unwind_tables=0')
 
+  if gyp_vars_dict.get('OS') == 'win':
```

Comment:
<code>enable_widevine</code> is a variable which is defined in <code>src/third_party/widevine/cdm/widevine_cdm.gyp</code> not <code>src/build/common.gypi</code>. Seems that I can't add it into <code>build/common.gypi</code> since this file hasn't recognized this variable. Am I correct?(Sorry that I am not familiar with gyp)